dist/Experiment/enums.d.ts

```diff
@@ -38,3 +38,19 @@ export declare enum EXPERIMENT_LOG_TYPE {
     EXPERIMENT_STATE_CHANGED = "experimentStateChanged",
     EXPERIMENT_DELETED = "experimentDeleted"
 }
+export declare enum SEARCH_KEY {
```

Comment:
Bro can we name these 
EXPERIMENT_SEARCH_KEY
EXPERIMENT_SORT_KEY
EXPERIMENT_SORT_AS

Reply:
Updated